dev/effect.nix

```diff
@@ -6,38 +6,15 @@
         # using the drv path here avoids downloading the closure on the deploying machine
```

Comment:
Cachix wants store paths instead of drvPath from the looks of it: https://docs.hercules-ci.com/hercules-ci-effects/reference/nix-functions/runcachixdeploy

But unsafeDiscardStringContext might also avoid the issue on those.

Reply:
I think needing the closure on the deploying machine is a blocker for switching to cachix deploy, it'll make it extremely slow.

Reply:
@domenkozar Does `cachix deploy activate` require the full closure?

Reply:
It needs the top-level derivation string for the node. It does expect that the node will be able to substitute the derivation and it's full closure.